raiden/tests/utils/smartcontracts.py

```diff
@@ -53,6 +53,7 @@ def deploy_tokens_and_fund_accounts(
 def deploy_contract_web3(
         contract_name: str,
         deploy_client: JSONRPCClient,
+        num_confirmations: typing.Optional[int] = None,
```

Comment:
`mypy` automatically infers the `Optional` type when the default value is set to `None`. So `num_confirmations: int = None` gives the same result here.